src/data/mv_polynomial/invertible.lean

```diff
@@ -0,0 +1,38 @@
+/-
+Copyright (c) 2020 Johan Commelin and Robert Y. Lewis. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin and Robert Y. Lewis
+-/
+
+import data.mv_polynomial.basic
+import algebra.invertible
+
+/-!
+# Invertible polynomials
+
+This file is a stub containing some basic facts about invertible elements in the ring of polynomials.
+-/
+
+open mv_polynomial
+
+/-- A natural number that is invertible when coerced to `ℚ` is also invertible
+when coerced to any `ℚ`-algebra. -/
```

Comment:
```suggestion
/-- A natural number that is invertible when coerced to `S` is also invertible
when coerced to any `S`-algebra. -/
```
Where `S` is a commutative semiring?  (Just thinking of obvious generalizations.)